### PR TITLE
Add new javascript question

### PR DESCRIPTION
**Pull request by Kirti2198**
Add one new javascript question

### CODE REVIEW

File: javascript/answers/q6.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What will be the output of the code below?">
```
Comment (arpansac) [inline javascript/answers/q6.html]: Please change the content of the meta tag to question text

**Review by arpansac — CHANGES_REQUESTED** [review]
Reviewed

**Comment by dakshkhetan** [comment]
@Kirti2198 Any updates? This PR will be closed in 2 days.

**Comment by Kirti2198** [comment]
I will do it today itself.

File: javascript/answers/Kirti2198-q6.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <div>
+            <div>
+                <h4>
+                    What is DOM? What is the use of document object?
+                </h4>
+            </div>
+            <div>
+                <strong>Answer:</strong> 
+                <p>DOM stands for Document Object Model.
+                  A document object represents the HTML document. It can be used to access and change the content of HTML.</p>
```
Comment (arpansac) [inline javascript/answers/Kirti2198-q6.html]: Please add an example!

**Review by arpansac — DISMISSED** [review]

File: javascript/answers/q6.html (hunk 1 above)
Comment (Kirti2198) [reply javascript/answers/q6.html]: Add meta  to question text

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**